src/shop/__init__.py

```diff
@@ -0,0 +1,35 @@
+import typing
+
+from src.shop.customer import Customer
+from src.shop.order import Order
+from src.shop.item import Item
+from src.shop.product import *
+
+
+def notify_everyone(orders: typing.List[Order]):
+    for order in orders:
+        order.customer.notify()
+
+def place_order(customer, *items):
+    '''
+    # customer jako obiekt klasy, items jako lista [produkt, ilość"
+    - nie umiem tego zapisać poprzez typowanie
```

Comment:
Lista będzie problematyczna w tym miejscu, ponieważ najlepiej jest wykorzystywać listę do trzymania elementów jednego typu. Proponowałbym użycie krotki. Wtedy definicja funkcji mogłaby wyglądać następująco:
```python
def place_order(customer: Customer, *items: typing.Tuple[Product, int]) -> Order:
    ...
```

Ciekawostka: zauważ, że mimo iż `items` będzie dla nas zmienną, po której iterujemy, to jako typ określamy jeden `item`, który zostanie przekazany jako argument pozycyjny.